examples/threelang/tfgrid/explorer.md

```diff
@@ -0,0 +1,32 @@
+# Explorer Example
+
+- This example uses the tfgrid explorer to retrieve information about the grid
+
+!!tfgrid.explorer.nodes
+    network: dev
+    count: true
+    node_id: 33
+    size: 100
+    page: 1
+
+!!tfgrid.explorer.farms
+    network: test
+    count: true
+    free_ips: 3
+
+!!tfgrid.explorer.contracts
+    network: qa
+    count: true
+    state: Created
+    size: 10
+    page: 5
+
+!!tfgrid.explorer.node
+    node_id: 1234
+
+!!tfgrid.explorer.stats
```

Comment:
What is this one? The overal statistics? Like ram, cpus, etc?

Reply:
yes. it includes info about nodes, farms, twins, contracts, node capacities, etc...